prow/cluster/cert-manager/tls-ing.yaml

```diff
@@ -1,16 +1,22 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
+  namespace: default
   name: tls-ing
   annotations:
     kubernetes.io/ingress.global-static-ip-name: prow
     kubernetes.io/tls-acme: "true"
+    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
+    certmanager.k8s.io/acme-http01-edit-in-place: "true"
     kubernetes.io/ingress.class: "gce"
 spec:
   tls:
   - secretName: prow-tls
+    hosts:
+    - prow.istio.io
   rules:
-  - http:
+  - host: prow.istio.io
```

Comment:
do we need others like http://velodrome.istio.io?

Reply:
not on the prow cluster. This ingress redirecting Prow services. Velodrome is hosted by k8s, so we have to use a proxy.